src/main/java/com/fasterxml/uuid/Generators.java

```diff
@@ -128,10 +128,8 @@ public static TimeBasedEpochGenerator timeBasedEpochGenerator()
 
     /**
      * Factory method for constructing UUID generator that generates UUID using
-     * version 7 (time+random based), using specified Ethernet address
-     * as the location part of UUID.
-     * No additional external synchronization is used.
-     */
+     * variant 7 (Unix Epoch time+random based).
```

Comment:
I assume this was accidental change, since it reverts "variant -> version" change from #68. Could you undo it? (unless this is intentional)